docs/setup.md

```diff
@@ -13,23 +13,23 @@ So, here are the steps:
 
  * install the dependencies:
 ```
-sudo apt install postgresql-11-postgis-2.5 slapd ldap-utils apache2 ca-certificates tomcat9 tomcat9-user postfix git
+sudo apt install postgresql-14-postgis-3 slapd ldap-utils apache2 ca-certificates tomcat9 tomcat9-user postfix git
```

Comment:
checking into a debian:bullseye docker image, it seems that the currently packaged version in this release is postgresql 13 (but postgis 3 indeed)

Reply:
hum yes and maybe we should remove the `sudo` part